napari/utils/colormaps/vendored/cm.py

```diff
@@ -39,7 +39,7 @@
 def _reverser(f, x=None):
     """Helper such that ``_reverser(f)(x) == f(1 - x)``."""
     if x is None:
-        # Returning a partial object keeps it picklable.
+        # Returning a partial object keeps it pickable.
```

Comment:
```suggestion
        # Returning a partial object keeps it pickeable.
```
Pretty sure that's what was intended here
https://en.wiktionary.org/wiki/pickleable

Reply:
You're right. I replaced a typo by a misspelling.